course/include/Isometry.hpp

```diff
@@ -93,43 +93,64 @@ inline std::ostream& operator<<(std::ostream& os, const Vector3& vect) {
 }
 
 class Matrix3 {
+    using mat3 = std::vector<std::vector<double>>;
+
    public:
-    Matrix3(const double m00, const double m01, const double m02, const double m10, const double m11, const double m13,
-            const double m21, const double m22, const double m23){
-                values_.push_back(m00);
-                values_.push_back(m01);
-                values_.push_back(m02);
-                values_.push_back(m10);
-                values_.push_back(m11);
-                values_.push_back(m13);
-                values_.push_back(m21);
-                values_.push_back(m22);
-                values_.push_back(m23);
-            }
+    // Constructors
+    Matrix3(const double m00, const double m01, const double m02, const double m10, const double m11, const double m12,
+            const double m20, const double m21, const double m22);
     Matrix3(const std::initializer_list<double>& row_1, const std::initializer_list<double>& row_2,
-            const std::initializer_list<double>& row_3){
-                for(auto i : row_1){
-                    values_.push_back(i);
-                }
-                for(auto i : row_2){
-                    values_.push_back(i);
-                }
-                for(auto i : row_3){
-                    values_.push_back(i);
-                }
-            }
+            const std::initializer_list<double>& row_3);
+    Matrix3();
 
     // Operators
     bool operator==(const Matrix3& a) const;
-    
+    bool operator==(const std::initializer_list<double>& rhs) const;
+    Matrix3 operator+(const Matrix3& a) const;
+    Matrix3 operator-(const Matrix3& a) const;
```

Comment:
For the sake of practice also create a move assignment operator and a copy assignment operator.